setup.py

```diff
@@ -9,66 +9,15 @@
 import os
 import sys
 
-try:  # for pip >= 10
-    from pip._internal.req import parse_requirements
-except ImportError:  # for pip <= 9.0.3
-    from pip.req import parse_requirements
```

Comment:
Have you researched the `pip` version on the various systems that we support?

Reply:
I think this will break `pip install kolibri` on a default Debian/Raspbian system.

https://packages.debian.org/stable/python-pip

Reply:
where is this used? the only place I saw was inside an `if False` block

Reply:
Yeah, it was added on the assumption that we would eventually do non-static pip builds for Kolibri, I guess?

Reply:
Oh right, didn't see it was made redundant further down, please continue! :+1: